src/services/getUsers.ts

```diff
@@ -8,11 +8,11 @@ import dummyUsers from './dummyData/users';
 export const getUsers = ({
   offset = 0,
   limit = 20,
-  query,
-  sortField,
-  sortOrder,
-  groupsId,
-  appsId,
+  query = '',
```

Comment:
Do we want an empty string for these? _.isNil won't filter them out.

Reply:
ah crap, typescript was yelling at me for not providing this 'required' attribute

:(

Reply:
Fixed! defaulting to null instead